src/unix/sentry_unix_path.c

```diff
@@ -104,6 +104,17 @@ sentry__path_join_str(const sentry_path_t *base, const char *other)
     return sentry__path_from_str_owned(sentry__stringbuilder_into_string(&sb));
 }
 
+sentry_path_t *
+sentry__path_clone(const sentry_path_t *path)
+{
+    sentry_path_t *rv = SENTRY_MAKE(sentry_path_t);
+    if (!rv) {
+        return NULL;
+    }
+    rv->path = sentry__string_dup(path->path);
```

Comment:
Just noticing this, can we rename that function to `sentry__string_clone` for consistency?